### PR TITLE
Add pouretrebelle to the about page

### PR DESCRIPTION
**Pull request by pouretrebelle**
## Changes

- Use greyscale css filter on avatars
- Add me! (I could have spent many more hours writing answers to those questions, I'm not happy with them and never will be)
- Update list of team members in the README

### CODE REVIEW

**Comment by pouretrebelle** [comment]
Huxley

File: src/components/AvatarShape.js [hunk 1]
```diff
@@ -31,6 +33,12 @@ const Image = styled('img')`
   position: absolute;
   top: 0;
   left: 0;
+  transition: filter ${TRANSITION_STYLE};
+  ${({$greyscale}) =>
+    $greyscale &&
+    `
+      filter: grayscale(1);
+  `}
```
Comment (pouretrebelle) [inline src/components/AvatarShape.js:41]: I can't see a way of doing this with styled-system without installing [the css package](https://styled-system.com/css) 🤔 is this okay?

File: README.md [hunk 2]
```diff
@@ -22,9 +22,17 @@
 We're working on updating our websites, domains, and npm package names so that it will be easier in future to find all things Primer. We welcome feedback and contributions at any time. Please check existing issues and pull requests before contributing, and please communicate via an issue first if you intend to make large-scale changes. See [contributing](.github/CONTRIBUTING.md) for more info.
 
 #### Team
+- [@pouretrebelle](https://github.com/pouretrebelle)
```
Comment (rezrah) [inline README.md:25]: thanks for updating this 🙏

File: src/components/MemberContainer.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ export default function Member(props) {
   const {member, isOdd, shape, ...rest} = props
   const colorName = member.color || 'blue'
   const color = themeGet(`colors.${colorName}.4`)(rest)
-  const avatarImage = hover ? member.gif : member.avatar
+  const avatarImage = hover ? member.gif : `https://avatars.githubusercontent.com/${member.handle}?s=600`
```
Comment (rezrah) [inline src/components/MemberContainer.js:28]: 💥 Nice. Do we always want this to be the GitHub avatar though? I think some of the team may have been uploading custom images intentionally for this in the past. Not sure though... cc. @yaili

File: src/components/AvatarShape.js [hunk 4]
```diff
@@ -77,18 +85,23 @@ const ShapeOutline = styled(Link)`
...
+      />
       <Shape shape={props.shape} />
     </A>
   </ShapeOutline>
 )
 
 AvatarShape.defaultProps = {
-  bg: 'blue.4'
+  bg: 'blue.4',
+  greyscale: false,
```
Comment (rezrah) [inline src/components/AvatarShape.js:99]: Should this be conditional? I think you can make it greyscale by default, as per the current site, which would remove the need for conditional css and extra props

File: src/data/team.js [hunk 5]
```diff
@@ -1,12 +1,21 @@
 const teamContent = [
+  {
+    name: 'Charlotte Dann',
+    title: 'Accessibility Software Engineer',
+    handle: 'pouretrebelle',
+    questionOne: `I get so much joy from writing HTML and CSS, and design systems are one level up - systematically making UIs accessible and consistent. I love concentrating on the tiny things like button states so product folk can build the big picture, knowing that the details are taken care of.`,
```
Comment (rezrah) [inline src/data/team.js:6]: ❤️

File: src/components/AvatarShape.js (hunk 4 above)
Comment (pouretrebelle) [reply src/components/AvatarShape.js:99]: The greyscale is turned off on hover so the gifs have colour:

| without greyscale | with greyscale |
|--------|--------|
| Screenshot 2021-11-25 at 14 25 45 | Screenshot 2021-11-25 at 14 26 04


I think ideally `AvatarShape` should be called `ShapeMask` and take a `children` prop for the avatar img so we don't have to pass the `greyscale` and `src` down (it's only used on this page) but I was wary of making bigger changes to this codebase (especially as I was only meant to be adding myself 😬)

File: README.md (hunk 2 above)
Comment (pouretrebelle) [reply README.md:25]: 👍 Added it to onboarding project templates

File: src/components/MemberContainer.js (hunk 3 above)
Comment (yaili) [reply src/components/MemberContainer.js:28]: is this for the about page? If so I'd say yes, give folks the chance to choose.

File: src/components/AvatarShape.js (hunk 4 above)
Comment (rezrah) [reply src/components/AvatarShape.js:99]: Yeah, I see the issue. I agree you'll need to forward a prop, so how about an alternative name of `hover` to give it more utility inside the component?

File: src/components/AvatarShape.js (hunk 1 above)
Comment (rezrah) [reply src/components/AvatarShape.js:41]: This is fine, although I think if you change the name to `hover` it's a bit easier to understand what's happening IMO

```scss
  filter: grayscale(1); // makes it grayscale by default

  ${({hover}) => hover && `filter: grayscale(0);` } // turn it off when hover is enabled for full colour gifs
```

File: src/components/AvatarShape.js (hunk 4 above)
Comment (rezrah) [reply src/components/AvatarShape.js:99]: Update: if you [try this](https://github.com/primer/primer.style/pull/279/files#r756989100), should fix this issue 👍

File: src/components/AvatarShape.js [hunk 6]
```diff
@@ -77,18 +82,20 @@ const ShapeOutline = styled(Link)`
 const AvatarShape = props => (
   <ShapeOutline shape={props.shape} bg={props.bg} {...props}>
     <A className={props.className} shape={props.shape}>
-      <Image src={props.src} />
+      <Image src={props.src} $hover={props.hover} />
```
Comment (rezrah) [inline src/components/AvatarShape.js:85]: Do you need $ prefix here? ☝️

**Review by rezrah — APPROVED** [review]
Looks great @pouretrebelle ... 🥳  🚢

File: src/components/AvatarShape.js (hunk 6 above)
Comment (pouretrebelle) [reply src/components/AvatarShape.js:85]: The $ turns it into a [transient prop](https://styled-components.com/docs/api#transient-props) so it's not output as an attribute on the DOM element. I just realised that we're not on styled v5.1 so this won't make a difference right now 😅  but when we do upgrade the package it'll make the html a teeny bit more valid.

Comment (rezrah) [reply src/components/AvatarShape.js:85]: Nice! TIL 🙌

**Pull request merged**